interface/patient_file/addr_label.php

```diff
@@ -45,37 +55,51 @@
 $today = oeFormatShortDate($date='today');
 $dob = oeFormatShortDate($patdata['DOB']);
 
-$pdf = new PDF_Label('5160'); // used this to get the basic info to the class
-$pdf = new eFPDF('P', 'mm',array(102,252)); // set the orentation, unit of measure and size of the page
-$pdf->AddPage();
-$pdf->SetFont('Arial','',50);
+
+//Format envelope settings.  Changes made by Daniel Pflieger
+//Pull the settings from globals.  If the Global settings have not been determined in Globals,
+//the default is to print on a 4 1/8 by 9 1/2 envelope.
+//todo: put the option in settings to set envelope size.
```

Comment:
looks like above comment (the last 3 lines) block isn't really pertinent. In order to ensure users have all the globals(so you don't need to worry about what to do if not set), I increment the $v_database in version.php whenever I add new globals. this then flags to run sql_upgrade.php in admin.php (for folks that are using the master branch), so new globals get brought in via sql_upgrade.php run (rather than having to save globals within OpenEMR to bring them in).